README.md

```diff
@@ -36,3 +51,15 @@ pip install -e .
 # Test adapter
 otioconvert -i some_timeline.aaf -o some_timeline.ext
 ```
+
+If you are using a version of OpentimelineIO that still has the AAF contrib adapter you may need to add the path of [plugin_manifest.json](./src/otio_aaf_adapter/plugin_manifest.json) to your `OTIO_PLUGIN_MANIFEST_PATH` [environment variable.](https://opentimelineio.readthedocs.io/en/latest/tutorials/otio-env-variables.html) This should override the contrib version.
+
+Contributions
+-------------
+
+If you have any suggested changes to the otio-aaf-adapter,
+please provide them via [pull request](../../pulls) or [create an issue](../../issues) as appropriate.
```

Comment:
This really works? If so I just learned something!

Reply:
yes it does, saw some other repos doing it.  Was very surprised to that it works!